### ISSUE TITLE
Unable to re-enable disabled rule

### ISSUE THREAD

**Issue by rawthey** [johngodley/redirection#813]
### Summary
Unable to delete or re-enable redirection rules which have been disabled 

### Actual Behavior

Clicking on either 'Delete' or 'Enable' for a disabled rule has no effect. There are no messages in the JavaScript console

### Steps to reproduce
I've confirmed that the problem only relates to rules which have been disabled, I am able to edit and delete active rules.

### Environment
Wordpress 4.9.4–en_GB,
Redirection 3.2
I see the same behaviour with Firefox 58.0.2 and Chrome 63.0.3239.132

**Comment by johngodley** [comment]
Can you try the 3.2.1 beta and see if it helps?

https://github.com/johngodley/redirection/releases/tag/3.2.1

**Comment by rawthey** [comment]
Just tried 3.2.1 beta but it didn't make any difference.

**Comment by johngodley** [comment]
Can you open your browser's developer tools, click on the 'network' tab, then try an delete something. You should a request being made in the network tab. Click on this request to show more information. Can you include that here?

**Comment by rawthey** [comment]
I've tried that but no request was generated.

This makes it look like the click handler has been disabled and made me suspect a conflict with some other plugin. After a bit of testing I've narrowed it down to a conflict with the  [Easy Forms for MailChimp](https://wordpress.org/plugins/yikes-inc-easy-mailchimp-extender/) plugin.

I can edit, delete and disable active rules but can't do anything with disabled rules so it looks like the MailChimp plugin is turning off click handlers for objects with class=row-action if they are contained in an object with class=disabled.
